package.json

```diff
@@ -75,6 +77,7 @@
     "gulp-rename": "^1.2.2",
     "gulp-run": "^1.7.1",
     "gulp-text-simple": "^0.4.1",
+    "italia-utils": "^0.0.3",
```

Comment:
I'm not mad about the naming. This is a general purpose package to generate Typescript schemas for io-ts, maybe that would be useful to not tie this to the italia* ecosystem as this could be an autonomous open source project by itself.

Reply:
yes agreed, I thought about that and realized that [there's still some stuff](https://github.com/teamdigitale/italia-utils/blob/master/templates/model.ts.njk#L120) that is very specific to our projects - we can definitely work on this and make it self-contained and reusable, find it a name and publish it by its own - right now I just needed something quick 😄

Reply:
that's right, we should make another repo for the custom types as well : )

Reply:
or perhaps contribute to https://github.com/gcanti/io-ts-types